exporter.py

```diff
@@ -154,15 +156,25 @@ def __init__(self):
                       self.bbs[len(self.bbs) - 1]['start_time']).total_seconds()
                      if self.bbs else 0)
         )
+        self.last_backup_size = Gauge('walg_last_backup_size',
```

Comment:
https://prometheus.io/docs/practices/naming/
...should have a suffix describing the unit, in plural form. Note that an accumulating count has total as a suffix, in addition to the unit if applicable.